.zuul/playbooks/containerd-build/run.yaml

```diff
@@ -0,0 +1,25 @@
+- hosts: all
+  become: yes
+  roles:
+    - role: config-golang
+      arch: arm64
+  tasks:
+    - name: Build containerd
+      shell:
+        cmd: |
+          set -xe
+          apt-get update
+          apt-get install -y btrfs-tools libseccomp-dev git pkg-config
+          echo $PATH
+          echo $GOPATH
+          mkdir -p $GOPATH/src/github.com/containerd/
+          ln -s '{{ ansible_user_dir }}/{{ zuul.project.src_dir }}' $GOPATH/src/github.com/containerd/containerd
+          cd $GOPATH/src/github.com/containerd/containerd
+          make | tee make.txt
+          make test | tee make_test.txt
```

Comment:
`make` and `make test` output will be shown in OpenLab job log file `job-output.txt.gz`, if you still need split logs, just need to redirect output to `{{ ansible_user_dir }}/workspace/logs`, see https://github.com/theopenlab/openlab-zuul-jobs/blob/master/roles/copy-files-on-node/tasks/main.yml#L9